crates/sui-framework/packages/sui-framework/tests/test_scenario_tests.move

```diff
@@ -1018,4 +1020,32 @@ module sui::test_scenario_tests {
         scenario.next_tx(sender);
         abort 42
     }
+
+    public struct E1(u64) has copy, drop;
```

Comment:
🎉

Reply:
Thanks for this feature, enjoyed not defining a dummy field :)